thavalon-webapp/src/components/gameComponents/playerBoard.tsx

```diff
@@ -66,8 +67,8 @@ export function PlayerBoard(): JSX.Element {
     const [missionResults, setMissionResults] = useState<MissionResultsMessage>();
     // State for maintaining the role of a player
     const [role, setRole] = useState<Role>(Role.Merlin);
-    // State for tracking the Agravaine player. This should merge into a map eventually for all declarations.
-    const [agravaine, setAgravaine] = useState<string>();
+    // State for tracking declared players. 
+    const [declarationMap, setDeclarationMap] = useState<Map<string, string>>(new Map());
```

Comment:
Based on the code below, does `declarationMap` store both Role -> Player and Player -> Role? That might cause problems if a player decides to call themselves Arthur or Agravaine

Reply:
Yes, this would be a problem. I've fixed it.